src/traces/indicator/attributes.js

```diff
@@ -327,6 +340,9 @@ module.exports = {
             description: 'Sets the width (in px) of the border enclosing the gauge.'
         },
         axis: overrideAll({
+            visible: extendDeep({}, axesAttrs.visible, {
```

Comment:
`extendFlat` should suffice here.

Reply:
Done in 95de45724b62a7a1c3d22434af838bd8c690cbfe